pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -136,23 +137,25 @@ export class Terminal extends React.Component {
         if (props.onTitleChanged)
             term.onTitleChange(props.onTitleChanged);
 
-        this.state = { terminal: term };
+        this.terminal = term;
+        this.state = {};
```

Comment:
This is at least missing `showPastingModal`. Would be good to have complete state here.

Reply:
Fixed.